packages/kotti-ui/source/kotti-field-number/utilities.ts

```diff
@@ -22,8 +22,8 @@ export const isInRange = ({
 	minimum,
 	value,
 }: {
-	maximum: KottiFieldNumber.Props['maximum']
-	minimum: KottiFieldNumber.Props['minimum']
+	maximum: KottiFieldNumber.PropsInternal['maximum']
```

Comment:
no consistency in those utils

sometimes, we make util generically support `number | null`

in this case, we rely on PropsInternal 

unify as generic `number | null`